metrics/network/iperf3_dockerfile/Dockerfile

```diff
@@ -3,13 +3,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Usage: FROM [image name]
-FROM fedora
+FROM fedora:34
```

Comment:
why this version? why not latest?

Reply:
If we use `latest` it will pick up Fedora 35 image. And I've seen dockerfile builds failing with Fedora 35 image just like below:

```
[root@xxxxx tests]# cd metrics/network/iperf3_dockerfile/
[root@xxxxx iperf3_dockerfile]# git diff
diff --git a/metrics/network/iperf3_dockerfile/Dockerfile b/metrics/network/iperf3_dockerfile/Dockerfile
index 2f6dce17..b7cc62f3 100644
--- a/metrics/network/iperf3_dockerfile/Dockerfile
+++ b/metrics/network/iperf3_dockerfile/Dockerfile
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Usage: FROM [image name]
-FROM fedora:34
+FROM fedora:35
 
 # Version of the Dockerfile
 LABEL DOCKERFILE_VERSION="1.0"
[root@xxxxx iperf3_dockerfile]# podman build -t iperf3 .
STEP 1: FROM fedora:35
STEP 2: LABEL DOCKERFILE_VERSION="1.0"
--> abe43026566
STEP 3: RUN dnf -y update &&     dnf -y install iperf3 &&     dnf clean all
Fedora 35 - x86_64                              0.0  B/s |   0  B     00:00    
Errors during downloading metadata for repository 'fedora':
  - Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-35&arch=x86_64 [getaddrinfo() thread failed to start]
Error: Failed to download metadata for repo 'fedora': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-35&arch=x86_64 [getaddrinfo() thread failed to start]
Error: error building at STEP "RUN dnf -y update &&     dnf -y install iperf3 &&     dnf clean all": error while running runtime: exit status 1

```